examples/src/main/java/io/druid/examples/ExamplesDruidModule.java

```diff
@@ -38,9 +39,8 @@
   {
     return Arrays.<Module>asList(
         new SimpleModule("ExamplesModule")
-            .registerSubtypes(
-                new NamedType(TwitterSpritzerFirehoseFactory.class, "twitzer")
-            )
+            .registerSubtypes(new NamedType(TwitterSpritzerFirehoseFactory.class, "twitzer"))
+            .registerSubtypes(new NamedType(GuavaDoubleMath.class, "guavaDoubleMath"))
```

Comment:
If you let me barge in, I agree with @fjy view which would make folks contribte expression functions easily when they come up with a new thing. However, it would be better to put that into a separate PR.